.github/workflows/focus-ios-ui-tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                xcodebuild_test_plan: ['SmokeTest', 'FullFunctionalTests']
+                xcodebuild_test_plan: ['SmokeTest'] # , 'FullFunctionalTests']
```

Comment:
Running all tests in both platforms exceeds the 4-hour timelimit imposed by Github Actions.